app/src/main/java/com/tari/android/wallet/model/TariWalletAddress.kt

```diff
@@ -83,9 +83,22 @@ data class TariWalletAddress(
     val interactive: Boolean
         get() = features.contains(Feature.INTERACTIVE)
 
+    // SafeTrade is a feature that allows the use of payment IDs in transactions.
+    val safetrade: Boolean
```

Comment:
```suggestion
    val payment_id_address: Boolean
```
I dont like the use of safetrade as a name for a feature here. 
Safetrade is an exchange, almost all exchanges will use this feature, not just them